app/controllers/declaration/DispatchLocationController.scala

```diff
@@ -39,8 +40,8 @@ class DispatchLocationController @Inject()(
 )(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport with ModelCacheable with SessionIdAware {
 
-  def displayPage(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
```

Comment:
Do we not need the async still given the updateCache returns a future?

Reply:
This is just `displayPage` - submit form is still `async`